app/assets/javascripts/explore_collections.js

```diff
@@ -0,0 +1,24 @@
+$(document).on('turbolinks:load', function() {
+  setExploreCollectionsHeight()
+  $(window).on('resize', setExploreCollectionsHeight)
+})
+
+function setExploreCollectionsHeight() {
+  if (desktopWidth(window)) {
+    matchFacetsHeight()
+  } else {
+    setDefaultHeight()
+  }
+}
+
+function desktopWidth(w) {
+  return w.matchMedia("(min-width: 992px)").matches
+}
+
+function matchFacetsHeight() {
```

Comment:
This cuts off the text of the card and doesn't allow user to scroll to see the rest. If that's the intended behavior, that's fine. If not, then something like overflow scroll needs to be implemented.

<img width="729" alt="Screen Shot 2020-04-09 at 11 04 18 AM" src="https://user-images.githubusercontent.com/18330149/78915698-efa92200-7a51-11ea-97b9-7f223167b9d1.png">

Reply:
You may not have as many facets in your local instance as we do in production. As a safeguard, I added a check to make sure that the height of the facets panel is at least as tall as the minimum card height before changing the card height.